api-module-library/hubspot/api.js

```diff
@@ -658,17 +658,26 @@ class Api extends OAuth2Requester {
     // **************************   Pages   *****************************
 
     async getLandingPages(query=''){
+        const options = {
+            url: `${this.baseUrl}${this.URLs.landingPages}`,
+        };
+        if (query !== '') {
+            options.url = `${options.url}?${query}`
```

Comment:
@MichaelRyanWebber 

If you pass a `query` object into the `options`, [we have a function that breaks it down into a query param](https://github.com/friggframework/frigg/blob/1399d648bbb4719750fb9f499824a761adc0d5c9/packages/module-plugin/requester/requester.js#L35)

This may not be ideal, depending on how a query is formed, so 👍 to ignore or not use it.